packages/app/src/app/pages/Dashboard/Content/routes/Recent/DocumentationRow.tsx

```diff
@@ -80,41 +81,35 @@ export const DocumentationRow: React.FC = () => {
     });
   };
 
+  const buildItems = () => {
```

Comment:
Could be changed to `const items = DOCS.map` variable instead of function if we want.

Reply:
Great suggestion! Addressed it here: [d273581](https://github.com/codesandbox/codesandbox-client/pull/7369/commits/d27358144657bc873a762469e032fbeaf4caf497)